contrib/docker/Dockerfile

```diff
@@ -7,8 +7,12 @@
 ###############
 
 # Basic Go environment with git, SSL CA certs, and upx.
-# golang:1.17.1-alpine (linux/amd64)
-FROM golang@sha256:13919fb9091f6667cb375d5fdf016ecd6d3a5d5995603000d422b04583de4ef9 AS builder
+# The image below is golang:1.18.0-alpine (linux/amd64)
+# It's pulled by the digest (immutable id) to avoid supply-chain attacks.
+# To update to a new digest, you must first manually pull the new image:
```

Comment:
What do you think about adding a `Maintainer note: ` line prefix and indent the next two lines to make it clear these aren't instructions for the end user of the Dockerfile?